packages/neuron-ui/src/locales/zh.ts

```diff
@@ -9,6 +9,7 @@ export default {
     Send: '转账',
     Receive: '收款',
     History: '交易历史',
+    UnspentCells: '未使用Cells',
```

Comment:
Like commented above, I don't thin the wallet interface should expose cells data. For a wallet user, she/he only cares about that she/he could spend amount <= balance. No need to expose cells concept to user.